src/multi_vector_simulator/E1_process_results.py

```diff
@@ -942,3 +944,29 @@ def get_units_of_cost_matrix_entries(dict_economic, kpi_list):
         else:
             unit_list.append(kpi_cost_unit_dict[key])
     return unit_list
+
+
+def get_emissions(dict_asset):
+    r"""
+    Calculates the total emissions of the asset per year.
+
+    Parameters
+    ----------
+    dict_asset : dict
+        Contains information about the asset.
+
+    Notes
+    -----
+    Tested with:
+    - test_get_emissions()
```

Comment:
```suggestion
    - E1.test_get_emissions()
```

Reply:
oh there are many functions where the prefix `E1` etc. is not given, but will do so from now on